systemtest/src/test/resources/oauth2/postgres.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     spec:
       containers:
         - name: postgres
-          image: postgres:latest
+          image: ${POSTGRES_IMAGE}
```

Comment:
It's just a nit, but why you need `${}` there, when you anyway need to replace it whole `${POSTGRES_IMAGE}`
You can use just `POSTGRES_IMAGE`, right?

Reply:
I can use just POSTGRES_IMAGE if you want @im-konge.

Reply:
okay :) you can keep it then